src/web/app/components/session-edit-form/session-edit-form.component.ts

```diff
@@ -134,6 +134,35 @@ export class SessionEditFormComponent {
     });
   }
 
+  /**
+   * Triggers the change of the model when the submission opening date changes.
+   */
+  triggerSubmissionOpeningDateModelChange(field: string, date: DateFormat): void {
+    const minDate: DateFormat = this.minDateForSubmissionStart;
+    if (JSON.stringify(minDate) === JSON.stringify(date)) {
```

Comment:
not a huge fan of this although it works, could we add a DateFormat comparision function in DateTimeService and use that here? (do add unit tests for the comparision function as well!)